components/Create.js

```diff
@@ -272,9 +302,25 @@ export default function Create({ user }) {
                             name={names.split(/\r?\n/)[currentCert - 1]}
                             style={{ width }}
                         />
-                    ) : (
-                        <p>Invalid certificate template</p>
-                    )}
+                    ) : certTemp === "Web Development Bootcamp" ? (
+                        <CertificateTemplate4
+                            id={certCode}
+                            title={title}
+                            line1={line1}
+                            line2={line2}
+                            line3={line3}
+                            issueDate={issueDate}
+                            signature={signature}
+                            leadUniversity={leadUniversity}
+                            date={date}
+                            name={names.split(/\r?\n/)[currentCert - 1]}
+                            style={{ width }}
+                        />
```

Comment:
Nothing to do here. but I just wanted to show you that the certificate template doesn't make any use of the issueDate variable